tests/zenko_tests/node_tests/backbeat/ReplicationUtility.js

```
@@ -393,30 +428,44 @@ class ReplicationUtility {
                     })),
                 },
                 UploadId: uploadId,
-            }, next),
+            }))
+                .then(data => next(null, data))
+                .catch(next),
         ], err => {
             if (err) {
-                return this.s3.abortMultipartUpload({
+                return this.s3.send(new AbortMultipartUploadCommand({
                     Bucket: bucketName,
                     Key: objectName,
                     UploadId: uploadId,
-                }, () => cb(err));
+                })).then(() => cb(err)).catch(() => cb(err));
```

finally + silent error